general/press.rst

```diff
@@ -191,8 +191,8 @@ Logos and Graphic material
 .. raw:: html
 
   <div align="center" class="align-center">
-    <img alt="../_downloads/openwisp-logo.svg"
-         src="../_downloads/openwisp-logo.svg"
+    <img alt="../../assets/design/openwisp-logo.svg"
```

Comment:
also notice that we use the image path as the `alt` attribute (alternative text), looks like a mistake, we can change it to `openwisp`

Reply:
Oh I thought that it was intentional. I added alt attribute it other two logos too. But, should be use alt as openwisp for all three logos or I should proceed like `Openwisp Black Logo` in alt